package.json

```diff
@@ -195,6 +195,7 @@
     "polyfill-object.fromentries": "^1.0.1",
     "popper-max-size-modifier": "^0.2.0",
     "qs": "^6.9.4",
+    "rc-slider": "^10.0.1",
```

Comment:
@aballerr used a different slider library for Price range, can two confirm which slider library we want to use moving forward?

Reply:
created a pr that changes the price range slider to use rc-slider